specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json

```diff
@@ -0,0 +1,313 @@
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Quotas.json#/definitions/QuotaList"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Quota": {
+            "description": "Holds quota information used to control resource allocation.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Quota properties.",
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/QuotaProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "../2015-12-01-preview/Compute.json#/definitions/Resource"
+                }
+            ]
+        },
+        "QuotaProperties": {
+            "description": "Properties for a Quota",
+            "type": "object",
+            "properties": {
+                "availabilitySetCount": {
+                    "description": "Maximum number of availability sets allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "coresLimit": {
+                    "description": "Maximum number of core allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "virtualMachineCount": {
+                    "description": "Maximum number of virtual machines allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "vmScaleSetCount": {
+                    "description": "Maximum number of scale sets allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "maxAllocationStandardManagedDisksAndSnapshots": {
+                    "description": "Maximum number of managed disks and snapshots of type standard allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "maxAllocationPremiumManagedDisksAndSnapshots": {
```

Comment:
Same as maxAllocationStandardManagedDisksAndSnapshots.

Reply:
same as above. consistent with syntax as per the other parameters. unless this is blocking don't want to change.